source/parser.hera

```diff
@@ -6300,6 +6300,8 @@ Reset
       verbose: false,
     }
 
+    module.config = parse.config = Object.assign({}, DEFAULT, parse.config)
```

Comment:
Will this interact correctly with virtual attributes like `coffeeConfig`? I worry it will just overwrite them, but perhaps not.

Reply:
fixed in d4ef779 👍